Translation - Improve German translation

OverlordZorn opened the pull request:
🇩🇪
got *slightly* sidetracked while working on the german guide 😅

OverlordZorn committed:
Update stringtable.xml

German Language Refinement

OverlordZorn committed:
Giftgas -> Senfgas

OverlordZorn committed:
Update stringtable.xml

OverlordZorn committed:
Merge remote-tracking branch 'upstream/dev-Tomcat' into dev-Tomcat

OverlordZorn committed:
German ChemWarfare Tripwire Update

OverlordZorn committed:
Merge branch 'dev-Tomcat' of https://github.com/PulsarNeutronStar/KAM into dev-Tomcat

OverlordZorn committed:
Merge branch 'Tomcat-SG:dev-Tomcat' into dev-Tomcat

OlafZigel committed:
Deutsch 05.03.2024

OlafZigel committed:
Updated Translation

OlafZigel committed:
Updated Translation V3

MiszczuZPolski committed:
fix indent

MiszczuZPolski committed:
fix duplicate

OlafZigel committed:
Update Translation V4

fixed indent [addons/pharma/stringtable.xml] line: 1111

deleted duplicate [addons/airway/stringtable.xml] line: 1330

OlafZigel committed:
Merge branch 'Translation-German' of https://github.com/OlafZigel/KAM into Translation-German

OlafZigel committed:
Updated Translation V5

OlafZigel committed:
Updated Translation V6

OlafZigel committed:
Updated Translation V7

OlafZigel committed:
Merge branch 'dev-Tomcat' into Translation-German

OverlordZorn committed:
:de:

OverlordZorn committed:
Merge branch 'dev-Tomcat' into translation-ger-hotfix

OverlordZorn committed:
missed some

MissHeda commented:
Why did you open a new translation PR when the is currently still one pending for German translation?
Won't this one interfere with the one that is already open?

OverlordZorn commented:
> Why did you open a new translation PR when the is currently still one pending for German translation? Won't this one interfere with the one that is already open?

thats not merged yet?

also, `Atemwegsverlegungswahrscheinlichkeit` triggered me

I dont mind reiterating over my PR once that one gets resolved.

OlafZigel commented:
> > Why did you open a new translation PR when the is currently still one pending for German translation? Won't this one interfere with the one that is already open?
> 
> thats not merged yet?
> 
> also, `Atemwegsverlegungswahrscheinlichkeit` triggered me
> 
> I dont mind reiterating over my PR once that one gets resolved.

If you want we could update my PR with the problems you fixed :D
Because I can't understand what the problem is

OlafZigel committed:
Updated Translation V8

OlafZigel committed:
Updated Translation V9

Idk what why it has changed the Japanese Translation. I hope it is still right :smiley:

OverlordZorn committed:
Merge branch 'pr/493' into translation-ger-hotfix

OverlordZorn committed:
Update stringtable.xml

OverlordZorn committed:
Update stringtable.xml

OverlordZorn commented:
I have merged @OlafZigel 's PR #493 into this PR, picking the best of both worlds.
I have merged tomcat-dev into this PR beforehand, meaning any accidental changes of the Japanese Lines should be reverted.

Afterwards, I double checked the files with tabler, no errors occour, meaning there are no duplicate entries left.


```
@MiszczuZPolski 
I dont know whats going on here:
![grafik](https://github.com/KAT-Advanced-Medical/KAM/assets/56258612/3ac5e61b-8641-409c-81a4-8ae415588098)

My local version seems fine
![grafik](https://github.com/KAT-Advanced-Medical/KAM/assets/56258612/00dd2188-e548-4d49-8c2a-da6597eddeb9)

dev-tomcat version:
https://github.com/KAT-Advanced-Medical/KAM/blob/b244f4360ac88d2631d88abfb81d6d2ad788f6a5/addons/circulation/stringtable.xml#L2047-L2051
```
Nvm, had an old version of KAM on my fork. now its not complaining anymore.

OverlordZorn committed:
Update stringtable.xml

OverlordZorn committed:
Merge remote-tracking branch 'upstream/dev-Tomcat' into translation-ger-hotfix

OverlordZorn committed:
indents

OverlordZorn committed:
Update stringtable.xml

OverlordZorn committed:
Update stringtable.xml

OverlordZorn committed:
Update stringtable.xml

OverlordZorn committed:
Update stringtable.xml

OlafZigel commented on addons/pharma/stringtable.xml:
```diff
@@ -2754,7 +2756,7 @@
             <French>Chance de blocage des IV dû à une administration intra-veineuse de médicaments sans rinçage à la solution saline</French>
             <Korean>식염수 주입 없이 약물 투여로 인한 IV 막힘 확률</Korean>
-            <German>Möglichkeit einer Verstopfung der IV durch Medikamentengabe ohne Kochsalzspülung</German>
+            <German>Möglichkeit einer Okklusion der IV durch Medikamentengabe ohne Kochsalzspülung</German>
```
Es ist eine Verstopfung des IV. Also würde ich das auch dabei belassen :D

OverlordZorn replied:
is das nich von dir?

OlafZigel replied:
Kann sein.. bitte ändere das um :D

OverlordZorn committed:
Update stringtable.xml

OverlordZorn commented:
resolve #493
